src/go/k8s/tests/e2e/superusers-prefix/01-set-superusers-prefix.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- command: kubectl patch deployment redpanda-controller-manager -n redpanda-system --type='json' -p='[{"op":"add", "path":"/spec/template/spec/containers/0/args/-", "value":"--superusers-prefix=__redpanda_cloud__"}]'
+- command: kubectl get deployment redpanda-controller-manager -n redpanda-system -o json
```

Comment:
Due to changes in 
https://github.com/redpanda-data/redpanda/pull/6942/files#diff-b41e7079a210a959c4f445bd5d8fd210184dd83aaf1af52617fcdadea041567aR29
https://github.com/redpanda-data/redpanda/commit/ff966e34d79c95641826392b02be3c54475d8428
This test should be moved to `unstable-test` as we resolve this problem of running e2e tests in parallel.

Reply:
Updated